### PR TITLE
[mirrororch]: Remove mirror session state after it is remvoed

### PR DESCRIPTION
**Pull request by stcheng**
Remove deprecated mirror session states in the state database

Signed-off-by: Shu0t1an Cheng <shuche@microsoft.com>

### CODE REVIEW

**Comment by stcheng** [comment]
retest this please

**Comment by stcheng** [comment]
retest this please

File: orchagent/mirrororch.cpp [hunk 1]
```diff
@@ -436,6 +438,13 @@ void MirrorOrch::setSessionState(const string& name, const MirrorEntry& session,
     m_mirrorTable.set(name, fvVector);
 }
 
+void MirrorOrch::removeSessionState(const string& name)
+{
+	SWSS_LOG_ENTER();
+
+	m_mirrorTable.del(name);
```
Comment (lguohan) [inline orchagent/mirrororch.cpp]: tab to spaces

File: orchagent/mirrororch.cpp [hunk 2]
```diff
@@ -113,17 +113,17 @@ bool MirrorOrch::bake()
...
+            // Recover saved active session's monitor port
+            m_recoverySessionMap.emplace(
+                    key, monitor_port + state_db_key_delimiter + next_hop_ip);
+        }
 
-        // Recover saved active session's monitor port
-        m_recoverySessionMap.emplace(
-                key, monitor_port + state_db_key_delimiter + next_hop_ip);
+	removeSessionState(key);
     }
```
Comment (lguohan) [inline orchagent/mirrororch.cpp]: tab to spaces

**Comment by lguohan** [comment]
can you describe the effort if a deprecated mirror session is not removed from state db? does it impact the warm reboot?

**Comment by stcheng** [comment]
@lguohan it doesn't impact the warm reboot; it could clean up the deprecated information or extra information stored before the warm reboot.

**Comment by stcheng** [comment]
Indentations are fixed.

**Pull request merged**